src/components/editor/markdown-preview/markdown-preview.tsx

```diff
@@ -41,7 +42,12 @@ const MarkdownPreview: React.FC<MarkdownPreviewProps> = ({ content }) => {
     md.use(markdownItRegex, replaceYouTubeLink)
     md.use(markdownItRegex, replaceVimeoLink)
     md.use(markdownItRegex, replaceGistLink)
-    md.use(MarkdownItParserDebugger)
+    md.use(markdownItContainer, 'success', { render: createRenderContainer('success') })
+    md.use(markdownItContainer, 'danger', { render: createRenderContainer('danger') })
+    md.use(markdownItContainer, 'info', { render: createRenderContainer('info') })
+    md.use(markdownItContainer, 'warning', { render: createRenderContainer('warning') })
+
+    // md.use(MarkdownItParserDebugger)
```

Comment:
Is disabling the parser-debugger plugin intended?

Reply:
no. reenabled